File: docs-site/ios/pub_stripped.swift
let success = try await realtime?.publish(topic: "<YOUR TOPIC>", message: [
    "user_name": "John Doe",
    "message": "How's it going fam?"
])

let success = try await realtime?.publish(topic: "<YOUR TOPIC>", message: "String messages are valid!")

// So are numbers
let success = try await realtime?.publish(topic: "<YOUR TOPIC>", message: 1234)

let success = try await realtime?.publish(topic: "<YOUR TOPIC>", message: 1234.123)

File: docs-site/android/pub_stripped.kt
val message = buildMap<String, Any> {
    put("user_name", "John Doe")
    put("message", "How's it going fam?")
}

val sent = client.publish("<YOUR TOPIC>", message)

val sent = client.publish("<YOUR TOPIC>", "String messages are valid!")

// So are numbers
val sent = client.publish("<YOUR TOPIC>", 1234)

val sent = client.publish("<YOUR TOPIC>", 1234.24)
